src/CLICommand/parameters_helper.ts

```diff
@@ -19,6 +19,14 @@ import passwordPrompt from 'prompts';
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type ParameterOptions = any;
 
+interface GetDriveKeyParams {
+	driveId: DriveID;
+	drivePassword?: string;
+	useCache?: boolean;
+}
+
+const staticDriveKeyCache: { [key: string]: Buffer } = {};
```

Comment:
I would make it a `private static` attribute of ParametersHelper rather than a global-scoped constant

Reply:
Also, TypeScript does allow us to use correct here, it's only with the key that it gets mad
```suggestion
const staticDriveKeyCache: { [key: string]: DriveKey } = {};
```